### PR TITLE
feat: add across protocol

### PR DESCRIPTION
**Pull request by liuyepiaoxiang**
## Checklist

- [x] I have followed the [Contributing Guidelines](../CONTRIBUTING.md)
- [x] (optional) As a contributor, my Ethereum address/ENS is:

### CODE REVIEW

**Comment by immasandwich** [comment]
Hello sir! Happy to see your contribution! Feel free to swing by the Discord in the #build-on-zapper channel for any questions, we'll be glad to support!

**Comment by liuyepiaoxiang** [comment]
lious.ethlious.eth

**Comment by liuyepiaoxiang** [comment]
lious.eth

File: src/apps/across/ethereum/eth.across.lp-fetcher.ts [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+import { Network } from '~types/network.interface';
+
+import { AcrossContractFactory } from '../contracts';
+import ACROSS_DEFINITION from '../across.definition';
+
+const appId = ACROSS_DEFINITION.id;
+const network = Network.ETHEREUM_MAINNET;
+
+@Register.TvlFetcher({ appId, network })
+export class EthereumAcrossTvlFetcher implements TvlFetcher {
```
Comment (immasandwich) [inline src/apps/across/ethereum/eth.across.lp-fetcher.ts:15]: Can you rename this file: `src/apps/across/ethereum/eth.across.lp-fetcher.ts` to `src/apps/across/ethereum/across.tvl-fetcher.ts`

Also, if there are no balances to be reported for the users, please delete all of the `across.balance-fetcher.ts` files.

**Comment by immasandwich** [comment]
1. The address returned by the l1Token call is in checksum format, lowercase it before you try to match on baseTokens
2. In your for loop on lines 53-56, you're not using the value. Use total += value instead of total += total

**Review by immasandwich — COMMENTED** [review]
Sorry for the delay ser! Reviewing now!

**Pull request merged**